zebra-state/src/service/finalized_state/disk_format/transparent.rs

```diff
@@ -123,9 +133,7 @@ impl IntoDisk for OutputLocation {
 
 impl FromDisk for OutputLocation {
     fn from_bytes(disk_bytes: impl AsRef<[u8]>) -> Self {
-        let hash_len = transaction::Hash::fixed_byte_len();
```

Comment:
I think this was probably the reason why we were using `fixed_byte_len`.

Reply:
Yep, but that will get removed in the next database change anyway.